### DISCUSSION TITLE
Could NOT find LAPACK (missing: LAPACK_LIBRARIES) CMake Error at CMakeLists.txt:155 (message):

### DISCUSSION THREAD

**Discussion by 123xyl** [openmopac/mopac#99] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/96686347/183245260-411af160-ae30-433b-8e9d-2989a35bb693.png)

**Comment by susilehtola** [comment]
You don't have BLAS/LAPACK development headers installed.

Also, it is dangerous to compile software as root.

**Reply by 123xyl** [reply]
got it,thank you very much

**Comment by godotalgorithm** [comment, marked as answer]
Some general advice about this. CMake has a system for automatically detecting BLAS and LAPACK libraries that are installed in standard locations and/or are in your system path. If you install some form of BLAS and LAPACK correctly, then this feature should work without any user intervention. If there is a problem, then you can provide the path to the BLAS and LAPACK libraries by hand with CMake command-line options. As stated in the error message, these would be something like

`cmake .. -DAUTOBLAS=OFF -DMOPAC_LINK="blas_and_lapack_directory_path"`
